partiql-plan/src/main/kotlin/org/partiql/plan/Operator.kt

```diff
@@ -7,5 +7,19 @@ public interface Operator {
 
     public fun <R, C> accept(visitor: Visitor<R, C>, ctx: C): R
 
+    /**
+     * Get i-th child (input) operator.
+     *
+     * @param index
+     */
+    public fun getChild(index: Int) {
+        throw UnsupportedOperationException("getChild")
+    }
```

Comment:
self-review: get i-th child is crucial to building the match structures but is unused now.